update: OpenSearch data migration prerequisite (#496)
aiven/aiven-docs: docs/products/opensearch/howto/migrate-snapshot-data-opensearch.md

- Optional: To verify the compatibility of your snapshot for migration, run
  the `pre_snapshot_checks.py` script from the
  [Aiven examples GitHub repository](https://github.com/aiven/aiven-examples/blob/main/solutions/validate-elasticsearch-to-opensearch-migration/pre_snapshot_checks.py).

> - Optional: If you want to verify the migration using our Script-based Validation

Reviewer: ```suggestion
- Optional: To verify the migration using Aiven's script-based validation
```

>   below, you need to take the initial data of the source cluster into a json file

Reviewer: ```suggestion
, put the initial data of the source cluster into a JSON file
```

>   before the data migration. Taking the data by running the

Reviewer: ```suggestion
  before the data migration. Get the data by running the
```

>   `get_migration_validation_data.py` script from the [[Aiven examples GitHub repository](https://github.com/aiven/aiven-examples/blob/main/solutions/validate-elasticsearch-to-opensearch-migration/get_migration_validation_data.py).

Reviewer: ```suggestion
  `get_migration_validation_data.py` script from the [[Aiven examples GitHub repository](https://github.com/aiven/aiven-examples/blob/main/solutions/validate-elasticsearch-to-opensearch-migration/get_migration_validation_data.py):
```

```bash
python get_migration_validation_data.py --patterns "YOUR_INDEX_PATTERNS" --waitsec 30 --outfile file1.json --es_host
```

> - Optiona: If you want to verify the migration using our Script-based Validation

Reviewer: Is this different from the migration validation steps above it? I see it includes the target cluster here, but it's not clear to me when you would use one over the other.
